test/common/upstream/BUILD

```diff
@@ -171,6 +176,16 @@ envoy_cc_test(
         "//test/test_common:simulated_time_system_lib",
         "//test/test_common:test_runtime_lib",
         "//test/test_common:utility_lib",
+    ],
```

Comment:
ideally the dependencies for the test library are exactly what needs to be included, and nothing more. I think a lot of these might belong to the test, could you fix up the best you can based on the includes you needed in the header/source? https://docs.bazel.build/versions/master/build-ref.html#actual_and_declared_dependencies

Reply:
Yup - you're right, forgot to change things here. Doing so now.

Reply:
Changed it. Let me know if the change looks good to you.

Reply:
Okay, just saw your last response. Did not add the declarations needed in test class, only in .h class. I need to add them back into build.